owners/src/notifier.js

```diff
@@ -89,7 +89,7 @@ class OwnersNotifier {
       ([name, filenames]) => {
         const header = `Hey @${name}, these files were changed:`;
         const files = filenames.map(filename => `- ${filename}`);
-        return [header, ...files].join('\n');
+        return [header, '```', ...files, '```'].join('\n');
```

Comment:
Nit: We probably don't need to prefix the filesnames with `-` anymore?

Reply:
Very true; that also makes this map wayyy simpler. Done.